packages/react/src/primitives/Table/TableHead.tsx

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames';
+import * as React from 'react';
+
+import { View } from '..';
+import { ComponentClassNames } from '../shared/constants';
+import { TableHeadProps } from '../types/table';
+
+export const TableHead: React.FC<TableHeadProps> = ({
```

Comment:
```suggestion
export const TableHead: Primitive<TableHeadProps, 'thead'> = ({
```
You may want to import `Primitive` from `../types/view`